src/client/src/ui/analytics/views/analytics.scss

```diff
@@ -33,87 +33,107 @@ $default-padding: 10px;
   padding: 10px;
 }
 
+.analytics__header--bold{
+  font-weight: bold;
+}
+
+.analytics__header--block{
+  display: block;
```

Comment:
-- is used to identify state rather than subcomponent. In this case, should just be analytics__header-block ;)